src/auth.rs

```diff
@@ -63,10 +63,10 @@ impl Authenticator {
 
         let start_doc =
             doc! {
-            "saslStart" => 1,
-            "autoAuthorize" => 1,
-            "payload" => binary,
-            "mechanism" => "SCRAM-SHA-1"
+            "saslStart": 1,
```

Comment:
Ugh, rustfmt, why...

I _think_ rustfmt won't change it back if the indentation is fixed here, but since this seems like it will happen every time, I'm fine with leaving it if you don't want to bother.

Reply:
I tested the latest rustfmt (0.9.0) and it fixed this issue, so I'm going to leave it for now and keep just the relevant macro changes.

Reply:
Sounds good; as mentioned before "latest rustfmt" is kind of a weird thing right now due to them restarting the versioning a few months back, so I think we should stick to 0.9.0 (the last one before restarting the versioning, which is what I'm using as well).